### PR TITLE
[REF] requirements.txt: Using valid pyyaml and jinja versions for py2 and py3

### PR DESCRIPTION
**Pull request by moylop260**
- [REF] tox.ini: Fix false red using git-submodules

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
-PyYAML
-jinja2
+PyYAML==3.12 ; python_version < '3.7'
```
Comment (luisg123v) [inline requirements.txt:1]: 3.7 -> 3?

File: tox.ini [hunk 2]
```diff
@@ -77,7 +77,8 @@ deps =
 skip_install = true
 commands =
     python setup.py check --strict --metadata --restructuredtext
-    check-manifest {toxinidir}
+    ; We are using git submodule in the package
+    ; check-manifest {toxinidir}
```
Comment (luisg123v) [inline tox.ini:81]: Thanks a lot for this!

File: requirements.txt (hunk 1 above)
Comment (moylop260) [reply requirements.txt:1]: Really
It was extracted from odoo/odoo
This version has issues only using py3.7

Why do you think that should be used for all py3 instead?

Comment (luisg123v) [reply requirements.txt:1]: Ok, I though it was a typo.

**Pull request merged**